<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRange</class>
 <widget class="QDialog" name="DialogRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>620</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabelClick" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>960</width>
     <height>680</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioH">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Highest Point</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioM">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>140</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Point</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioL">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>220</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lowest Point</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="textH">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textM">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>170</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textL">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>250</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelH">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>90</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelM">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>170</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelL">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>250</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="posLabelH">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>120</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="posLabelM">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>200</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="posLabelL">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>280</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="posLabelTest">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>410</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioTest">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>380</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="cdButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>450</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture Frame</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textH_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>90</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textH_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>90</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textM_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>170</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textM_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>170</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textL_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>250</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textL_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>250</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelClick</class>
   <extends>QLabel</extends>
   <header location="global">qlabelclick.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogRange</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogRange</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
